recipe/meta.yaml

```diff
@@ -11,27 +11,36 @@ source:
 
 build:
   noarch: python
+  skip: true # [py<37]
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
     - poetry-core >=1.0.0
-    - python >=3.7
+    - python
+    - wheel
   run:
-    - python >=3.7
+    - py >=1.11.0
+    - python
     - werkzeug >=2.0.0
 
 test:
+  imports:
+    - coverage
   source_files:
     - tests
   requires:
     - pip
     - pytest >=7.1.3
+    - pytest-cov
     - requests >=2.28.1
+    - types-requests >=2.28.9
+    - pre-commit 3.2.1
+    - mypy
     - toml
-    - types-toml >=0.10.8
+    - types-toml >=0.10.8.6
```

Comment:
Upstream uses `^0.10.2`, which can be translated to `>=0.10.0,<0.11.0` or `0.10.*`. See https://python-poetry.org/docs/dependency-specification/#caret-requirements.

Also, why are some dependency following what's defined in upstream and some don't?